cmd/emp/destroy.go

```diff
@@ -49,3 +46,10 @@ func runDestroy(cmd *Command, args []string) {
 		}
 	}
 }
+
+func warningMessage(args []string) (warning, desired string) {
+	appname := args[0]
```

Comment:
I addressed this in the other comment, but this would panic if you don't provide an app name, since the `len(args)` check happens within runDestroy.